src/chrome/content/rules/kernsec.org.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+	Invalid certificate:
+        www.kernsec.org
+    
+    Redirect to HTTP:
```

Comment:
I cannot reproduce this (and it would also break the ruleset). We use this term to avoid loops if the server responses with a 3xx or JavaScript to force plain HTTP.

Reply:
I'm not sure what you mean. Is the comment wrong? If so, what should it be?

Reply:
Yes, its wrong. It does not redirect back to plain http:
```
$  curl -IL https://kernsec.org/
HTTP/1.1 301 Moved Permanently
Date: Mon, 23 Sep 2019 22:41:20 GMT
Server: Apache/2.2.15 (CentOS)
Location: https://kernsec.org/wiki/
Content-Type: text/html; charset=iso-8859-1

HTTP/1.1 301 Moved Permanently
Date: Mon, 23 Sep 2019 22:41:20 GMT
Server: Apache/2.2.15 (CentOS)
X-Powered-By: PHP/5.3.3
X-Content-Type-Options: nosniff
Vary: Accept-Encoding,Cookie
Expires: Thu, 01 Jan 1970 00:00:00 GMT
Cache-Control: private, must-revalidate, max-age=0
Last-Modified: Mon, 23 Sep 2019 22:41:20 GMT
Location: https://kernsec.org/wiki/index.php/Main_Page
Content-Type: text/html; charset=utf-8

HTTP/1.1 200 OK
Date: Mon, 23 Sep 2019 22:41:20 GMT
Server: Apache/2.2.15 (CentOS)
X-Powered-By: PHP/5.3.3
X-Content-Type-Options: nosniff
Content-language: en
Vary: Accept-Encoding,Cookie
Expires: Thu, 01 Jan 1970 00:00:00 GMT
Cache-Control: private, must-revalidate, max-age=0
Last-Modified: Mon, 30 Oct 2017 23:37:25 GMT
Content-Type: text/html; charset=UTF-8
```

Please just remove it (but keep the one for `www`).

Reply:
Done https://github.com/madaidan/https-everywhere/commit/35fe4c0c5f4daf0a92d71bf1606f823430d6c2c6

Reply:
Thanks!